src/main/java/uk/gov/companieshouse/client/ChipsRestClient.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.companieshouse.client;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.companieshouse.model.ChipsContact;
+
+@Component
+public class ChipsRestClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ChipsRestClient.class);
+
+    private final RestTemplate restTemplate;
+
+    @Autowired
+    public ChipsRestClient(RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    public ResponseEntity<String> createContactInChips(ChipsContact contactDetails, String chipsUri) {
```

Comment:
I'd recommend to make client return void / throw error in case non 2xx response is received cause clients responsibility is to hide HTTP related complexity.

Your solution works through so it is just gentle recommendation, not must have.

Reply:
I've made a change to the client to return void and throw error when non 2xx response